FirebaseRemoteConfig/Tests/SwiftAPI/APITests.swift

```diff
@@ -138,6 +152,159 @@ class APITests: APITestBase {
     waitForExpectations()
   }
 
+  func testFetchAndActivateUnchangedConfig() {
+    guard !APITests.useFakeConfig else { return }
```

Comment:
@ryanwilson Is `if ... return` preferable to `guard !`?

I find the double negative confusing ...

Reply:
Yup, I'll make those more clear.

Reply:
Hmm, I do prefer the `guard` in general for early exits... what do you think about `guard APITests.useFakeConfig == false else { return }`?

I don't have super strong feelings so I'm fine keeping it as `if` if someone does :)